annotation/tests/test_tasks_services.py

```diff
@@ -0,0 +1,52 @@
+from unittest.mock import Mock, patch
+
+import pytest
+from sqlalchemy.orm import Session
+
+from annotation.errors import FieldConstraintError
+from annotation.jobs.services import ValidationSchema
+from annotation.tasks.services import validate_task_info
+
+
+@pytest.fixture
+def db_session():
+    """Fixture for creating a mock database session."""
+    return Mock(spec=Session)
```

Comment:
Good use of Mock object. We should check if there exists a similar fixture inside of `conftest.py`, so that we don't duplicate code.

Reply:
There isn't something like this in conftest.py, so I created it as a @pytest.fixture to avoid code duplication because I will need the DB session in many tests. After writing the upcoming tests, I might extend this function further.